content/docs/components/hyperparameter-tuning/katib-config.md

```diff
@@ -114,7 +114,13 @@ All of these settings except **`image`** can be omitted. If you don't specify an
 
 1. `image` - Docker image name for the `random` suggestion.
 
-    **Must be specified**.
+    **Must be specified**. You can specify following images:
+
+    - `suggestion-chocolate`: [chocolate](https://github.com/AIworx-Labs/chocolate) based suggestion service which supports `grid`, `chocolate-random`, `chocolate-quasirandom`, `chocolate-bayesian-optimization` and `chocolate-mocmaes`.
```

Comment:
Maybe it is better to say: "Which supports algorithm names: `grid`, ..." ?
And do we need to say about each algorithm name. For example:
`chocolate-quasirandom`: Chocolate implementation of quasirandom algorithm.

Reply:
I mind two things about this:

* In the discussion of https://github.com/kubeflow/katib/pull/1131#discussion_r407955790, we decided to not introduce `goptuna-random` and `goptuna-tpe`. So I don't know how should I describe `tpe` and `random`.
* I think it's better to describe `image` field of suggestion settings in this section. How about describe algorithm name in other parts?